Gems/Automation/Code/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+ly_add_target(
```

Comment:
Can you make an `Automation.API` HEADERONLY target that has a list of header files?

```
ly_add_target(
    NAME Automation.API HEADERONLY
    NAMESPACE Gem
    FILES_CMAKE
        automation_api_files.cmake
    INCLUDE_DIRECTORIES
        INTERFACE
            Include
)
```